test/api/v3/integration/groups/POST-groups_groupId_join.test.js

```diff
@@ -225,4 +225,46 @@ describe('POST /group/:groupId/join', () => {
       });
     });
   });
+
+  context('Party incentive achievements', () => {
+    let user, invitedUsers, party;
+
+    beforeEach(async () => {
+      let { group, groupLeader, invitees } = await createAndPopulateGroup({
+        groupDetails: {
+          name: 'Test Party',
+          type: 'party',
+        },
+        members: 1,
+        invites: 3,
+      });
+
+      party = group;
+      user = groupLeader;
+      invitedUsers = invitees;
+    });
+
+    it('awards Party Up achievement to party of size 2', async () => {
+      await invitedUsers[0].post(`/groups/${party._id}/join`);
+
+      await expect(invitedUsers[0].get('/user')).to.eventually.have.deep.property('achievements.partyUp', true);
+      await expect(user.get('/user')).to.eventually.have.deep.property('achievements.partyUp', true);
+    });
+
+    it('does not award Party On achievement to party of size 2', async () => {
```

Comment:
I think this one is failing because it expects the property to be there but not to be true and in this case it's missing completely